engine/docs/INPUT_SYSTEM.md

```diff
@@ -0,0 +1,37 @@
+## Defold Input system
+
+### dmHID
+
+This library collects the information from the "Human Interface Devices".
+It may be a keyboard, mouse or a gamepad.
+
+Each platform has its own implementation of how the interaction is done with the hardware.
+
+### dmInput
+
+The input library uses the `dmHID` as the abstraction for the input.
+
+This library uses a `.gamepads` file for translating from the raw hardware gamepad information
+to how the Defold engine expects the data to be. E.g. which hardware button corresponds to which engine enum.

```

Comment:
I don't think we should add manual line breaks to this kind of documentation. A markdown renderer should be able to generate flowing text without it. Also when translating from markdown to html there's less risk that a newline like this gets turned into a `</br>`. This section of text should be rendered as a single `<p>`aragraph and the browser should handle text flow.